apps/frontend/src/components/ImageUpload.tsx

```diff
@@ -1,8 +1,18 @@
-import { useState, useEffect } from 'react';
+import React,{ useRef, useState, useEffect } from 'react';
+import Select from 'react-select';
+
+import { countryOptions } from '../countries_list';
+
+type ImageUploadProps = {
+  onSearch: (data: { country: string; imageUrl: string }) => void;
+};
+
+const ImageUpload: React.FC<ImageUploadProps> = ({ onSearch }) => {
```

Comment:
I find this ⬆️  syntax rather confusing.
This ⬇️  one's much more clear:
```suggestion
const ImageUpload = ({ onSearch }: ImageUploadProps) => {
```